tensorflow/java/BUILD

```diff
@@ -15,6 +15,16 @@ java_library(
     visibility = ["//visibility:public"],
 )
 
+# Operator convenience wrappers for the core tensorflow library.
+java_library(
+    name = "tensorflow_op",
```

Comment:
Instead of a separate `java_library`, how about including `:java_op_sources` in the `tensorflow` target? That way we end up with one `.jar`. Otherwise we have to worry about distribution of this jar and will have to deal with more complicated instructions on www.tensorflow.org/install/install_java

(The Android builds use the `java_sources` filegroup so they won't include these files).